pkg/database/rql/GPGKey_test.go

```diff
@@ -0,0 +1,402 @@
+package rql
+
+import (
+	"fmt"
+	"github.com/kylelemons/godebug/pretty"
+	"github.com/quan-to/chevron/pkg/models"
+	"github.com/quan-to/slog"
+	r "gopkg.in/rethinkdb/rethinkdb-go.v6"
+	"strings"
+	"testing"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).